Lms/canvas/backend/.env.sample

```
@@ -0,0 +1,4 @@
+CANVAS_API_BASE=https://canvas.instructure.com/api/v1/
+
+
+CANVAS_API_TOKEN=GEJcMkVKehnL9zxzCYD9fA4KuLvWDZzM3UfA3AvN2eQXFT3ZWMX2GE9ZMD64PeRz
```

Keep it just like this
```suggestion
CANVAS_API_TOKEN=
```